arm/ripper/main.py

```diff
@@ -142,20 +173,56 @@ def main(logfile, job):
         logging.info("Getting MakeMKV hashed keys for UHD rips")
         grabkeys()
 
+    #  Entry point for dvd
     if job.disctype in ["dvd", "bluray"]:
         # get filesystem in order
-        hboutpath = os.path.join(job.config.ARMPATH, str(job.title))
+        #  If we have a nice title/confirmed name use the MEDIA_DIR and not the ARM unidentified folder
+        if job.hasnicetitle:
+            #  Make sure we dont use 0000 in our folder name
+            if job.year != "0000":
+                hboutpath = os.path.join(job.config.MEDIA_DIR, str(job.title + " (" + job.year + ")"))
+            else:
+                hboutpath = os.path.join(job.config.MEDIA_DIR, str(job.title))
+        else:
+            hboutpath = os.path.join(job.config.ARMPATH, str(job.title))
 
```

Comment:
You do a check for duplicates below, but the check of duplicate is based on if the folder exists in the target destination.
Something I had on my future plans for this project was to have the scripts check to historical jobs to see if the CRC of the disc in play matches anything that already exists. If the CRC matches, use the title and year from that entry. 
(I wanted this functionality to reduce wasted time when I am repeatedly ripping the same disc while tweeking the HandBrake parameters (Before you start, I am aware that there may be problems with multiple matches).

It might be a more robust solution to check the jobs table for successful jobs with a matching CRC to qualify something as a dupe. From what I have seen a number of people use their destination directory as more of a 'staging ground' before it hit's their plex or emby server.

Reply:
The dupe check could be much better. The only reason i included this is that, it solves a number of issues.

- It solves the "not moving" issue (Bug where ARM will not move correctly identied movies)
- it solves the "duplicate jobs" issue  where jobs trigger more than once. (Bug/ possibly faulty drives issue)

But i agree that a CRC lookup with successful jobs would be a much better way of checking for if the disc is a dupe or not.

Reply:
I managed to make a much better check for dupe entries using the database.

For testing this make things sooo much easier. Ive written it so that even if dupe rips are disabled it will allow through rips until a successful rip is added to the database. Then it fully disables re-rips.
It also takes the confirmed details from the previous rips. 

So if you have a disc that doesnt get ident when using OMdb and you search/add details, all future dupes will try to use the details you have set.

I still need to test it more in VMs/main server. However it works great on my test server. Great idea!